package.json

```diff
@@ -4,11 +4,13 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "source ./.env && rm -rf build/ && tsc -p tsconfig.json && npm start",
+    "dev": "nodemon build/index.js",
```

Comment:
I've not run this locally, but I'd say development should happen against TS files, not JS files in `build` directory.

`nodemon --exec ts-node index.ts` is better. Otherwise, `nodemon` would run after `watch` script runs `tsc` and updates `build` directory. This can have issues with generated files, like for DB migration etc.